src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -302,12 +310,23 @@ export const ControlGroupStrings = {
           i18n.translate('controls.controlGroup.management.hierarchy.title', {
             defaultMessage: 'Chain controls',
           }),
-        getHierarchySubTitle: () =>
-          i18n.translate('controls.controlGroup.management.hierarchy.subtitle', {
+        getHierarchyTooltip: () =>
+          i18n.translate('controls.controlGroup.management.hierarchy.tooltip', {
             defaultMessage:
               'Selections in one control narrow down available options in the next. Controls are chained from left to right.',
           }),
       },
+      showApplySelections: {
+        getShowApplySelectionsTitle: () =>
+          i18n.translate('controls.controlGroup.management.showApplySelections.title', {
+            defaultMessage: 'Apply selections automatically',
+          }),
+        getShowApplySelectionsTooltip: () =>
+          i18n.translate('controls.controlGroup.management.showApplySelections.tooltip', {
+            defaultMessage:
+              'If disabled, control selections will only take effect when the apply button is clicked.',
+          }),
+      },
```

Comment:
Copy help

Reply:
@amyjtechwriter

Reply:
I think the repetition is ok, and that the sentence is the best one to explain to the user what they need to do. Lets go with your "If disabled, control selections will only be applied after clicking apply." alternative @Heenawter!

Reply:
Changed in https://github.com/elastic/kibana/pull/174714/commits/9fbe3f4b2af6bfb4d4937e65291b8c2ccac01c08 🙌 Thank you!!